providers/nodeReleasesDataProvider.tsx

```diff
@@ -0,0 +1,27 @@
+import { createContext } from 'react';
+import type { FC, PropsWithChildren } from 'react';
+import type {
+  NodeReleaseData,
+  NodeReleasesDataContext as NodeReleasesDataContextType,
+} from '../types';
+
+type NodeReleasesDataProviderProps = PropsWithChildren<{
+  releases: NodeReleaseData[];
+}>;
+
+export const NodeReleasesDataContext =
+  createContext<NodeReleasesDataContextType>(undefined as any);
```

Comment:
We shouldn't set this as `undefined`. The initial value needs to have the same shape. At maximum what you can do is create a similar shape: `{ lts: undefined, current: undefined, releases: [] }`